soccer/radio/SimRadio.cpp

```diff
@@ -134,7 +162,6 @@ void SimRadio::switchTeam(bool blueTeam) {
                                 .arg(blueTeam ? "blue" : "yellow")
                                 .toStdString());
     }
-    std::cout << "Bound to " << SimBlueStatusPort << std::endl;
```

Comment:
It was inconsistent with the below block that didn't have a printout and the value it was printing out should have been "RadioRxPort + _channel" instead of just the SimBlueStatusPort. I figured we didn't really need the printouts anyway.